package.json

```diff
@@ -47,20 +47,38 @@
   "devDependencies": {
     "@commitlint/cli": "11.0.0",
     "@commitlint/config-conventional": "11.0.0",
+    "@types/got": "9.6.11",
+    "@types/jquery": "3.5.5",
+    "@types/lodash.merge": "4.6.6",
+    "@types/mocha": "8.2.0",
```

Comment:
You should use 7.x for mocha types as we have to stick on 7.x because of nodejs 8.